Sources/Core/Oracle/GasOracle.swift

```diff
@@ -137,7 +137,7 @@ final public class Oracle {
             latestBlockNumber = block
         case let .exact(number): latestBlockNumber = number
             // Error throws since pending and erliest are unable to be used in this method.
-        default: throw Web3Error.valueError
+        default: throw Web3Error.valueError()
```

Comment:
Suggestions:
- remove `// Error throws since pending and erliest are unable to be used in this method.`
- change `valueError()` to `valueError(desc: "Unable to use '\(block)' policy to resolve block number to calculate gas fee suggestion.")`